hack/make-rules/make-help.sh

```diff
@@ -21,22 +21,21 @@ set -o pipefail
 readonly red=$(tput setaf 1)
 readonly reset=$(tput sgr0)
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
 ALL_TARGETS=$(make -C "${KUBE_ROOT}" PRINT_HELP=y -rpn | sed -n -e '/^$/ { n ; /^[^ .#][^ ]*:/ { s/:.*$// ; p ; } ; }' | sort)
-CMD_TARGETS=$(ls -l "${KUBE_ROOT}/cmd" |awk '/^d/ {print $NF}')
+CMD_TARGETS=$(cd "${KUBE_ROOT}/cmd"; find . -type d -mindepth 1 -maxdepth 1 | cut -c 3-)
```

Comment:
ls output is not reliably portable / not specified. this version should be more portable and doesn't require knowing awk 😉 

same as https://github.com/kubernetes/kubernetes/pull/79316 we're `cd`ing to do find relative to `.` and using `cut -c 3-` to strip the leading `./`

Reply:
`find $KUBEROOT/cmd -type d`

Reply:
if we do that we have to strip the leading `$KUBEROOT` which may be some arbitrary absolute path, which is more complex than stripping the leading `./`.